scripts/open_covid_data/open_covid_19_data.mcf

```diff
@@ -0,0 +1,972 @@
+medicalStatus: dcs:	PatientDeceased
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased
+typeOf: dcs:StatisticalVariable
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientDeceased
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedPatientDeceased
+typeOf: dcs:StatisticalVariable
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:PatientDeceased
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalConditionIncident_COVID_19_ConfirmedCase
+typeOf: dcs:StatisticalVariable
+incidentType: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+medicalStatus: dcs:ConfirmedCase
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_MedicalConditionIncident_COVID_19_ConfirmedCase
+typeOf: dcs:StatisticalVariable
+incidentType: dcs:COVID_19
+measuredProperty: dcs:cumulativeCount
+medicalStatus: dcs:ConfirmedCase
+populationType: dcs:MedicalConditionIncident
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalTest_ConditionCOVID_19
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_Person_COVID_19_Tested_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalTest_COVID_19_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:IncrementalCount_MedicalTest_COVID_19
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested_PCR
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+medicalTestingMethod: dcs:COVID19PCRTest
+statType: dcs:measuredValue
+
+Node: dcid:CumulativeCount_Person_COVID_19_Tested
+typeOf: dcs:StatisticalVariable
+medicalCondition: dcs:COVID_19
+measuredProperty: dcs:incrementalCount
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
```

Comment:
Do these two have the wrong pop type? Please tag me for a second review once this is sorted out.

Reply:
@tjann Can you please take a second look?

Reply:
I still see "dcid:CumulativeCount_Person_COVID_19_Tested" and "populationType: dcs:MedicalTest" -- is that what you see too?

Reply:
@eftekhari-mhs

Reply:
no it is changed to person